hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHiveIncrementalPuller.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.hudi.utilities;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestHiveIncrementalPuller {
+
+  private HiveIncrementalPuller.Config config;
+
+  @Before
+  public void init() {
+    config = new HiveIncrementalPuller.Config();
+  }
+
+  @Test
+  public void testInitHiveIncrementalPuller() throws Exception {
+
+    HiveIncrementalPuller puller = new HiveIncrementalPuller(config);
```

Comment:
Not following this.. a constructor can either throw an exception or return a non-null object right? should we be using a try-catch and calling `fail` from the catch block to ensure the NPE is not thrown?

Reply:
Got it, thank you for your guidance.

Reply:
Done.